### DISCUSSION TITLE
Feature Request: Only activate when specific exe's are running

### DISCUSSION THREAD

**Discussion by yamrod** [cafali/SnapKey#87] [New Features & Ideas]
I'd like to set this app to start on login but I don't want it to activate unless specific game exe processes are running so as not to interfere with other productivity tasks. Is there a way to do specify when it should be active based on what process is currently active or by other means?

**Comment by LightBeamOvHe** [comment]
I'll go even as far as to say that maybe it could be toggled automatically depending on the focused program ? With a whitelist or blacklist logic. Idk how feasible that is or if that would make the program more « resource hungry », but that could be nice
Like, when you're playing the game it's on, but if you alt tab on your browser for example, it's toggled off until you come back to the game
